main/lsp/notifications/sorbet_workspace_edit.cc

```diff
@@ -85,12 +85,15 @@ void SorbetWorkspaceEditTask::runSpecial(LSPTypechecker &typechecker, WorkerPool
     typechecker.state().epochManager->startCommitEpoch(updates->epoch);
     startedNotification.Notify();
     // Only report stats if the edit was committed.
-    if (!typechecker.typecheck(move(*updates), workers)) {
+    if (typechecker.typecheck(move(*updates), workers)) {
```

Comment:
Fixes a bug where we treated canceled slow paths as successful slow paths -- whoops!

Reply:
(This bug only impacts metrics, not correctness, FYI.)